### PR TITLE
Added additional templates to the project

### PR DESCRIPTION
**Pull request by sparuvu**
What was changed? Why is this necessary?
----------------------------------------

Updated the following templates 
* Releasing.md - Updated the release documentation with the guidelines for releasing a project.
* Contributors - added a thank note to all the contributors.

Added the following templates 
 * Code of Conduct - https://www.contributor-covenant.org/
 * Issue Template - Template for logging the issues
 * Code Owners - Automatically adds code owners as reviewers 
 * Stale Config - automatically closes the stale issues based on this configuration.
How was it tested?
----

This is just adding/updating the templates. No code changes involved. 

How to test
----

*This is bare minimum acceptable testing*

- [x] `mvn clean install -U`

Reviewers
----

- [ ] [John Leacox](https://github.com/johnlcox)
- [ ] [Sundeep Paruvu](https://github.com/sparuvu)
- [ ] [Nimesh Subramanian](https://github.com/nimeshsubramanian)
- [ ] [Supriya Lal](https://github.com/lal-s)
- [ ] [Brian van de Boogaard](https://github.com/b-boogaard)
- [ ] [Nathan Schile](https://github.com/shiftyeyes)

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 14
```
Comment (johnlcox) [inline .github/stale.yml]: I'm not sure we should auto close stale issues.

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -29,3 +33,5 @@
 [bryan-baugher]: https://github.com/bbaugher
 [eric-christensen]: https://github.com/EricChristensen
 [cal-fisher]: https://github.com/cbfshr
+
+Thanks to everyone who helped in building this Repository.
```
Comment (shiftyeyes) [inline CONTRIBUTORS.md]: nit. lower case Repository.

File: .github/stale.yml (hunk 1 above)
Comment (sparuvu) [reply .github/stale.yml]: if we don't close, issues might get piled up. Contributors can always reopen them or keep them active by making a comment.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [beadledom@cerner.com](mailto:beadledom@cerner.com). All
```
Comment (johnlcox) [inline CODE_OF_CONDUCT.md]: Does someone actually monitor this email address?

Comment (b-boogaard) [reply CODE_OF_CONDUCT.md]: I'm not sure I even know the credentials to check it.

Comment (sparuvu) [reply CODE_OF_CONDUCT.md]: All the core developers from Cerner are aliases of this email ID.

**Pull request merged**